drivers/segger/Make.defs

```diff
@@ -27,11 +27,11 @@ ifeq ($(CONFIG_SEGGER_RTT),y)
   CFLAGS += -Wno-shadow -Wno-array-bounds
 
   ifneq ($(CONFIG_SEGGER_RTT_SECTION),"")
-    CFLAGS += ${shell $(DEFINE) "$(CC)" SEGGER_RTT_SECTION=CONFIG_SEGGER_RTT_SECTION}
+    ${call DEFINE, CFLAGS, SEGGER_RTT_SECTION=CONFIG_SEGGER_RTT_SECTION}
```

Comment:
```suggestion
    ${call DEFINE,CFLAGS,SEGGER_RTT_SECTION=CONFIG_SEGGER_RTT_SECTION}
```
The `call` function arguments should not be preceded with whitespaces to prevent unwanted effects:
See the "final caution" at the end of the documentation at https://www.gnu.org/software/make/manual/html_node/Call-Function.html

Reply:
thanks~